public/app/plugins/panel/dashlist/styles.ts

```diff
@@ -25,15 +25,15 @@ export const getStyles = stylesFactory((theme: GrafanaTheme) => ({
   dashlistStar: css`
     display: flex;
     align-items: center;
-    color: ${theme.colors.textWeak};
+    color: ${theme.v1.colors.textWeak};
     cursor: pointer;
     z-index: 1;
   `,
 
   dashlistFolder: css`
-    color: ${theme.colors.textWeak};
-    font-size: ${theme.typography.size.sm};
-    line-height: ${theme.typography.lineHeight.sm};
+    color: ${theme.v1.colors.textWeak};
+    font-size: ${theme.typography.h6.fontSize};
+    line-height: ${theme.v1.typography.lineHeight.sm};
```

Comment:
This should be the same as `theme.typography.body.lineHeight` but you can try `theme.typography.h6.lineHeight` as well (it's 1.5 vs 1.6)